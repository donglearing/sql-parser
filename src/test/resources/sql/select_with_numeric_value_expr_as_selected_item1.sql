select t.a - t.b as MIN_FUNC, COUNT(t.c) from T as t group by MIN_FUNC