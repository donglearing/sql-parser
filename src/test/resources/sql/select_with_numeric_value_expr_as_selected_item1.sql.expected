SELECT t.a - t.b AS MIN_FUNC, COUNT(t.c) FROM T AS t GROUP BY MIN_FUNC